### PR TITLE
[tagger] Update docs

### PR DESCRIPTION
**Pull request by davidor**
### What does this PR do?

Updates the Tagger component documentation.

### Describe how to test/QA your changes

Skip.

### CODE REVIEW

File: comp/core/tagger/README.md [hunk 1]
```diff
@@ -62,31 +48,29 @@ cache. Cache invalidation is triggered by the collectors (or source) by either:
...
+**TagStore**.
 
 For convenience, the package creates a **defaultTagger** object that is used
 when calling the `tagger.Tag()` method.
 
-                   +-----------+
-                   | Collector |
-                   +---+-------+
+                   +--------------+
+                   | Workloadmeta |
```
Comment (adel121) [inline comp/core/tagger/README.md:72]: should this be `workloadmeta collector` instead? 

To me both seem ok, but `workloadmeta collector` is more informative.

WDYT?

File: comp/core/tagger/README.md [hunk 2]
```diff
@@ -62,31 +48,29 @@ cache. Cache invalidation is triggered by the collectors (or source) by either:
...
-|-----------------------------------------|--------------------------------------------------------------------|
-| workloadmeta.KindContainer              | `container_id://<sha>`                                             |
-| workloadmeta.KindContainerImageMetadata | `container_image_metadata://<sha>`                                 |
-| workloadmeta.KindGardenContainer        | `container_id://<sha>`                                             |
-| workloadmeta.KindKubernetesPod          | `kubernetes_pod_uid://<uid>`                                       |
-| workloadmeta.KindECSTask                | `ecs_task://<task-id>`                                             |
-| CloudFoundry LRP                        | `<processGuid>/<svcName>/<instanceGuid>`  or `<appGuid>/<svcName>` |
-| Container runtime or orchestrator       | (none)                                                             |
-| Kubernetes Endpoint                     | `kube_endpoint_uid://<namespace>/<name>/<ip>`                      |
-| Kubernetes Service                      | `kube_service://<namespace>/<name>`                                |
```
Comment (adel121) [inline comp/core/tagger/README.md:75]: why did some services disappear? were they removed from the code but not from documentation?

Comment (davidor) [reply comp/core/tagger/README.md:75]: I think so. Here's the current list of the objects handled by the tagger: https://github.com/DataDog/datadog-agent/blob/854f00295eab22f9d07851a66bfd847cc8eefc2e/comp/core/tagger/taggerimpl/collectors/workloadmeta_extract.go#L132 All of them come from workloadmeta

**Review by adel121 — APPROVED** [review]

**Pull request merged**